scss/main.scss

```diff
@@ -287,23 +287,55 @@ h1,h2,h3 {
   margin-bottom: 2rem;
   padding-left: 2rem;
   padding-right: 2rem;
+  p.text-center {
+    margin-bottom: 48px;
+  }
 }
 
 #meetupLink {
   color: $cfa-blue;
 }
 
+.meetup-content {
+  margin-bottom: 1.5rem;
+  > * {
+  margin-bottom: 1rem;
+  }
+}
+
+
+
 .rsvp-stuff {
   margin-top: 1rem;
+  p {
+    font-family: $font-family-alt;
+    font-weight: 500;
+    color: $darker-grey;
+    font-size: .875rem; 
+  }
 }
 
 .attendees {
   border-bottom: 2rem solid transparent;
 }
 
 .attend-button {
+  font-family: $font-family-alt;
+  font-weight: 500;
+  color: $darker-grey; 
+  font-size: .875rem;
   border-bottom: 1rem solid transparent;
   padding-left: 5%;
+  @media (max-width: 361px) {
```

Comment:
Can you please double-check this media query? The breakpoint should be a lot larger than 361px.

Reply:
Absolutely! Double-checking now.

Reply:
You were right, thank you! Joseph said 768px for now.